frontend/apps/filemanager/filemanager.lua

```diff
@@ -253,6 +253,18 @@ function FileManager:setupLayout()
                     text = _("Reset settings"),
                     enabled = is_file and DocSettings:hasSidecarFile(BaseUtil.realpath(file)),
                     callback = function()
+                        local other_buttons
+                        if DocumentRegistry:getProvider(file).provider == "crengine" then
```

Comment:
Why limit this to crengine documents ? Some cre docs may be opened with MuPDF, and vice-versa.